addons/TBMod_arsenal/arsenal/fn_arsenalUSA.sqf

```diff
@@ -937,6 +939,19 @@ _items append (switch (ACE_player getVariable ["TB_rolle", ""]) do
         [
             // Gewehre
             "rhs_weap_m14ebrri",
+            "rhs_weap_mk17_CQC",
+            "rhs_weap_mk17_LB",
+            "rhs_weap_mk17_STD",
+            "rhs_weap_sr25",
+            "rhs_weap_sr25_d",
+            "rhs_weap_sr25_ec_d",
+            "rhs_weap_sr25_ec_wd",
+            "rhs_weap_sr25_wd",
+
+            // Munition
+            "rhsusf_20Rnd_762x51_SR25_m62_Mag",
```

Comment:
Munition gehört nicht in die Rollen, sondern in die Mengen, damit der Munitionsträger diese Munition mitnehmen kann... ggf neue menge für Rifle machen und Muniträger geben.